genimage.c

```diff
@@ -473,6 +474,8 @@ static int collect_mountpoints(void)
 
 const char *mountpath(const struct image *image)
 {
+	if(image->srcdir) return image->srcdir;
```

Comment:
Space after `if` and `return ...` in a new line.

Reply:
done